src/coreclr/vm/comtoclrcall.cpp

```diff
@@ -489,7 +489,7 @@ extern "C" UINT64 __stdcall COMToCLRWorker(Thread *pThread, ComMethodFrame* pFra
     // idea is that we needn't really do this work either in static assembly code nor in dynamically
     // generated code since the benefit/cost ratio is low.  There are some minor differences in the below
     // code, compared to x86.  First, the reentrancy and loader lock checks are optionally compiled into the
```

Comment:
```suggestion
    // code, compared to x86.
```
These checks do not exist in CoreCLR